### PR TITLE
[14.0][FIX/IMP] account_payment_partner

### PR DESCRIPTION
**Pull request by alexis-via**
FIX form view of account.move
Replace python constraint by check_company=True on payment_mode_id of account.move

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check Travis

**Comment by StefanRijnhart** [comment]
Travis agrees that this change improves the consistency of the data :-)
```
Traceback (most recent call last):
  File "/home/travis/build/OCA/bank-payment/setup/account_payment_partner/odoo/addons/account_payment_partner/tests/test_account_payment_partner.py", line 297, in test_invoice_constrains
    "payment_mode_id": self.supplier_payment_mode_c2.id,
...
  File "/home/travis/OCB-14.0/odoo/models.py", line 3900, in create
    records._check_company()
  File "/home/travis/OCB-14.0/odoo/models.py", line 3321, in _check_company
    raise UserError("\n".join(lines))
odoo.exceptions.UserError: Incompatible companies on records:
- 'Draft Bill MISC/2020/12/0001' belongs to company 'YourCompany' and 'Payment Mode' (payment_mode_id: 'Suppliers Bank 2') elongs to another company.
 ```

**Comment by pedrobaeza** [comment]
No news on this?

**Comment by pedrobaeza** [comment]
@alexis-via are you going to fix this PR?

**Comment by alexis-via** [comment]
I updated the tests (raises UserError instead of ValidationError). The tests work on my laptop ; Travis should be green this time.

File: account_payment_partner/views/account_move_view.xml [hunk 1]
```diff
@@ -23,22 +23,18 @@
         <field name="model">account.move</field>
         <field name="inherit_id" ref="account.view_move_form" />
         <field name="arch" type="xml">
-            <xpath
-                expr="//div[field[@name='invoice_payment_term_id']]"
-                position="after"
-            >
+            <field name="partner_bank_id" position="before">
```
Comment (pedrobaeza) [inline account_payment_partner/views/account_move_view.xml:26]: You are moving this to a place that is not always shown (I don't see it in customer invoices) and not in main section. Please put it before `journal_id` instead.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by alexis-via** [comment]
@pedrobaeza  I don't understand your last remark. This is the view I have on customer invoices:

customer_invoice

**Comment by pedrobaeza** [comment]
I don't see anything with your debug mode activated and overlapping the screenshot. Can you repeat it?

**Comment by alexis-via** [comment]
Sorry about this bad screenshot. Here is a better one
fac_client2

**Comment by pedrobaeza** [comment]
I'm not seeing "Recipient Bank" in runbot, but OK if it's always show in that position.

/ocabot merge minor

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**